planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp

```diff
@@ -127,7 +127,7 @@ class NormalLaneChange : public LaneChangeBase
   std::vector<double> calc_prepare_durations() const;
 
   lane_change::TargetObjects getTargetObjects(
```

Comment:
change to use snake_case?

Reply:
done https://github.com/autowarefoundation/autoware.universe/pull/9085/commits/46cf31901739f76a0b699edd9b542012d140611e